src/org/labkey/test/tests/ClientAPITest.java

```diff
@@ -1448,47 +1452,61 @@ public void contentTypeResponseTest()
         // 30509: For 401 and 404, respond with same ContentType as request
         log("Testing response content type for 404");
         Connection cn = getConnection(true);
-        Command command = new Command("bam", "boozled");
+        Command<CommandResponse> command = new Command<>("bam", "boozled");
 
         Map<String, Object> expectedProps = new HashMap<>();
         expectedProps.put("success", false);
         expectedProps.put("exception", "No LabKey Server module registered to handle request for controller: bam");
 
-        runCommand(cn, command, "application/json", 404, expectedProps);
-        runCommand(cn, command, "text/xml", 404, expectedProps);
-        runCommand(cn, command, "text/html", 404, null);
+        runCommand(cn, command, "application/json", "text/html", 404, expectedProps);
+        runCommand(cn, command, "text/xml", "text/html", 404, expectedProps);
+        runCommand(cn, command, "text/html", "text/html", 404, null);
```

Comment:
This change seems to directly contradict the issue that it is testing for: https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=30509

Reply:
Indeed. I talked with @labkey-nicka and we agree that the new behavior is more correct than the old. We were using the request content type as a proxy for what the response should look like, since we didn't have a way to pipe the actual response format in from the action into ExceptionUtil until my change in the platform PR here.